### PR TITLE
Wallet: Better button styles

### PR DESCRIPTION
**Pull request by figitaki**
Adds all the different button styles as a `Button` component.
Screen Shot 2019-05-13 at 4 53 21 PM

### CODE REVIEW

File: frontend/wallet/src/render/components/Footer.re [hunk 1]
```diff
@@ -36,57 +34,24 @@ module StakingSwitch = {
...
-      <button
-        onClick={_e => {
-          let task =
-            Bindings.Navigator.Clipboard.writeTextTask(
-              PublicKey.toString(pubKey),
-            );
-          Task.perform(task, ~f=()
-            // TODO: Should we toast when this happens? Do we need to handle errors?
-            => Js.log("Copied to clipboard"));
-        }}
```
Comment (Schmavery) [inline frontend/wallet/src/render/components/Footer.re]: Good code to remember for when we reimplement this

**Comment by Schmavery** [comment]
Can we start adding screenshots to PRs for the more style-oriented changes?

File: frontend/wallet/src/render/Theme.re [hunk 2]
```diff
@@ -30,8 +30,9 @@ module Colors = {
   let yeezy = `hex("C53131");
 
   let midnight = `hex("1F2D3D");
-  let marineLight = `rgba((51, 104, 151, 0.1));
   let marine = `hex("336897");
+  let marineLight = `rgba((51, 104, 151, 0.1));
```
Comment (Schmavery) [inline frontend/wallet/src/render/Theme.re]: Is there a reason this doesn't use the same "xAlpha" pattern as other colors?

Comment (figitaki) [reply frontend/wallet/src/render/Theme.re]: Ah yeah that's a good idea.

**Pull request merged**